common/deps.bzl

```diff
@@ -5,28 +5,14 @@ def rules_python():
     git_repository(
         name = "rules_python",
         remote = "https://github.com/bazelbuild/rules_python.git",
-        tag = "0.1.0",
-        patches = [
-            # Force rules_python to export the requirements.bzl file in
-            # order for stardoc to be able to load it during documentation
-            # generation.
-            "@vaticle_bazel_distribution//:bazelbuild_rules_python-export-requirements-bzl-for-stardoc.patch",
-        ],
-        patch_args = ["-p1"],
+        tag = "0.24.0",
     )
 
 def rules_pkg():
-    http_archive(
+    git_repository(
```

Comment:
Note: as described additionally in https://github.com/vaticle/bazel-distribution/pull/378
We need to depend on the git repository so that we get access to stardoc rules, in addition to the rest of the rules.